annotator/pom.xml

```diff
@@ -33,6 +33,29 @@
       <artifactId>commons-cli</artifactId>
       <version>1.3</version>
     </dependency>
+    <dependency>
+        <groupId>javax.xml.bind</groupId>
+         <artifactId>jaxb-api</artifactId>
+        <version>2.3.0</version>
```

Comment:
i think because further down i disabled the configuration of this module using spring boot I needed to add this dependency